### PR TITLE
Domain and entity schema tests

### PR DESCRIPTION
**Pull request by jdorweiler**
Adding some basic schema tests for domains and entities. They're slow enough already so I'm only looking at the top level properties, not drilling down into the resources. 

Testing: `npm run test`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "description": "Data set of the most common third party domains on the web with information about their behavior, classification and ownership.",
+  "scripts": {
+    "test": "mocha --timeout 30000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/duckduckgo/tracker-radar.git"
+  },
+  "author": "DuckDuckGo",
+  "license": "CC-BY-NC-SA-4.0",
```
Comment (kdzwinel) [inline package.json:13]: Our license file says "CCBY-NC-SA 4.0" no idea if both of those are considered valid.

File: test/schema.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    "domain": {
+        "type": "object",
+        "properties": {
+            "domain": {"type": "string"},
+            "owner": {"type": "object"},
```
Comment (kdzwinel) [inline test/schema.js:6]: can we expand here to cover individual fields?

File: test/schema.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            "prevalence": {"type": "number"},
+            "fingerprinting": {"type": "integer", "minimum": 0, "maximum": 3},
+            "cookies": {"type": "number"},
+            "categories": {"type": "array"},
+            "default": {"type": "string"},
+            "rules": {"type": "array"}
```
Comment (kdzwinel) [inline test/schema.js:12]: ditto

File: test/schema.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    "domain": {
+        "type": "object",
+        "properties": {
+            "domain": {"type": "string"},
+            "owner": {"type": "object"},
+            "prevalence": {"type": "number"},
+            "fingerprinting": {"type": "integer", "minimum": 0, "maximum": 3},
+            "cookies": {"type": "number"},
+            "categories": {"type": "array"},
```
Comment (kdzwinel) [inline test/schema.js:10]: I remember that I saw you doing "array of strings" somewhere 🤔

File: test/schema.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+            "default": {"type": "string"},
```
Comment (kdzwinel) [inline test/schema.js:11]: can we do an enum with "block"/"ignore"?

File: test/schema.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+            "rules": {"type": "array"}
+        }
+    },
+    "entity": {
+        "type": "object",
+        "properties": {
+            "properties": {"type": "array"},
```
Comment (kdzwinel) [inline test/schema.js:18]: array of strings, right?

**Review by kdzwinel — COMMENTED** [review]
🤩 This is great, thanks for doing it! Can we be even more detailed in that schema? (I left some ideas)

File: package.json (hunk 1 above)
Comment (jdorweiler) [reply package.json]: Not sure, changed it so they match at least.

File: test/schema.js (hunk 5 above)
Comment (jdorweiler) [reply test/schema.js]: Removed 'default', not needed in here.

**Comment by jdorweiler** [comment]
Added more schema tests.

**Comment by kdzwinel** [comment]
So good 👏

**Pull request merged**